src/NHSD.BuyingCatalogue.Identity.Api/Views/Account/ForgotPassword.cshtml

```diff
@@ -10,6 +10,14 @@
         <div data-test-id="login-page">
             <div class="nhsuk-grid-row">
                 <div class="nhsuk-grid-column-two-thirds">
+                    <div data-test-id="go-back-section" class="nhsuk-back-link">
```

Comment:
you don't need `data-test-id` on every div - we just have the one below

Reply:
(unless you are using it in your tests)

Reply:
It's being used in the tests